_chapters/3.md

```diff
@@ -0,0 +1,830 @@
+---
+url: 3.html
+number: III
+title: Viewing Data in the Browser
+layout: rails_tutorial
+---
+
+{% sectionheader %}
+  Viewing Data in the Browser
+{% endsectionheader %}
+
+{% steps %}
+  {% list %}
```

Comment:
@alimi I'm going to merge this because I'm updating the tutorial for the new stuff but for future chapters don't use indentation - It breaks everything. I'll clean this one up since I'm already doing it.